### PR TITLE
Fix runner state in MessageProcessingSpecTest

### PR DESCRIPTION
**Pull request by MatheusFranco99**
This PR fixes the runner state for the CommitteeRunner test cases in the MessageProcessingSpecTest.

### CODE REVIEW

**Comment by GalRogozinski** [comment]
> Before, I have also tried using the test.Runner when creating a runner for a duty. But some tests had troubles

I thought it would be simple. If I am wrong and you don't manage to fix this way simply write an explanation of why 🙏

**Comment by MatheusFranco99** [comment]
@GalRogozinski 
Fixed.
The tests that had errors were due to using the same Runner for more than one duty. But these tests are now of type CommitteeSpecTest, so all solved :)

File: ssv/spectest/generate/state_comparison/tests_MultiMsgProcessingSpecTest/consensus_future_decided/sync committee.json [hunk 1]
```diff
@@ -110,8 +110,7 @@
 														0,
 														1,
 														2
-												],
-												"IsStopped": false
```
Comment (GalRogozinski) [inline ssv/spectest/generate/state_comparison/tests_MultiMsgProcessingSpecTest/consensus_future_decided/sync committee.json]: what's with this change 🤔

Comment (GalRogozinski) [reply ssv/spectest/generate/state_comparison/tests_MultiMsgProcessingSpecTest/consensus_future_decided/sync committee.json]: It seems that jsons were not generated correctly in #412

**Pull request merged**